apps/studio/components/interfaces/Settings/Logs/Logs.DatePickers.tsx

```diff
@@ -166,6 +169,7 @@ export const LogsDatePicker = ({
 
           setStartDate(fromDate)
           setEndDate(toDate)
+          setCurrentMonth(new Date(toDate))
```

Comment:
```suggestion
          setCurrentMonth(toDate)
```

Same suggestion here, unless I'm missing something